src/providers/node/mod.rs

```diff
@@ -83,9 +96,12 @@ impl Provider for NodeProvider {
 
     fn build(&self, app: &App, _env: &Environment) -> Result<Option<BuildPhase>> {
         let mut build_phase = BuildPhase::default();
+        let pkg_manager = NodeProvider::get_package_manager(app);
 
-        if NodeProvider::has_script(app, "build")? {
-            let pkg_manager = NodeProvider::get_package_manager(app);
+        if NodeProvider::is_nx_monorepo(app) {
+            let app_name = NodeProvider::get_nx_app_name(app, _env)?.unwrap();
+            build_phase.add_cmd(format!("{} run {}:build:production", pkg_manager, app_name));
```

Comment:
Should this be `npm nx run APP:build:production` since this script may not exist. I am seeing this error when building the default NX app.

<img width="1037" alt="image" src="https://user-images.githubusercontent.com/3044853/184257219-3e7d6f95-c7b5-4b47-87a5-2325e35c0fcf.png">

Reply:
I believe you are correct. I'm refactoring the build command so it works no matter what package manager the user is using. Just didn't get enough time last night  😆